third_party/sl4a/src/main/java/com/google/android/mobly/snippet/SnippetRunner.java

```diff
@@ -48,6 +49,9 @@
 
     @Override
     public void onCreate(Bundle arguments) {
+        // Prevent this runner from triggering any real JUnit tests in the snippet by feeding it a
+        // hardcoded empty test class.
+        arguments.putString("class", EmptyTestClass.class.getCanonicalName());
```

Comment:
Just a tad clearer.
Since you were gonna reassign it to mArguments, wouldn't it be better to use a consistent name in the method as opposed to two different names?